src/main/java/com/park/easyrecruit/servlet/application/Applications.java

```diff
@@ -21,28 +21,26 @@
  * @author andrei
  */
 @WebServlet(name = "Applications", urlPatterns = {"/Applications"})
-@ServletSecurity(value = @HttpConstraint(rolesAllowed = {"ClientRole"}))
+@ServletSecurity(value = @HttpConstraint(rolesAllowed = {"ClientRole","ManageInterviewRole"}))
 public class Applications extends HttpServlet {
 
     @Inject
     private ApplicationBean applicationBean;
 
-    /**
-     * Handles the HTTP <code>GET</code> method.
-     *
-     * @param request servlet request
-     * @param response servlet response
-     * @throws ServletException if a servlet-specific error occurs
-     * @throws IOException if an I/O error occurs
-     */
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
-        
+
```

Comment:
As a general rule in web development GET methods should never have any side effects, i.e. never change or update data. Please `POST` or `PUT` for this purpose.